### ISSUE TITLE
The search function isn't working

### ISSUE THREAD

**Issue by Pr0teus** [trendmicro/aleph#32]
When we search for any string in the "Search" field (left sidebar) does not bring any results.

**Comment by jseidl** [comment]
This works. Maybe not as everyone expects. It uses the elasticsearch query
language.

Most of the problem is that the mapping doesn't exists for plugins and such

**Comment by turicas** [comment]
You may use [ES Query DSL](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-query.html) or [Lucene query syntax](https://lucene.apache.org/core/2_9_4/queryparsersyntax.html). The [Python's ElasticSearch API documentation](https://elasticsearch-py.readthedocs.org/en/master/api.html#elasticsearch) may help also.
To test, you can open an IPython session and try it interactively:

```python
from aleph.datastore import es
es.search(...)
es.lucene_search(...)
```

**Comment by jseidl** [comment]
Exactly. For full text search the mapping must be implemented correctly. I
think we need documentation of the search function and also an whitelist
filter on the input.

Merces, since you're good with regex you could try to make this whitelist
rule ;)

**Comment by danilovazb** [comment]
I'll work with this issue.

**Comment by SLAYEROWNER** [comment]
me too.

**Comment by loganbr** [comment]
\0/

**Comment by merces** [comment]
I renamed the issue. Thank you all for working on this.

**Comment by danilovazb** [comment]
Whats's guys, all right?

I was editing at the settings.py file on the server made available at the lecture about Aleph, these lines are what?

ELASTICSEARCH_URI = "127.0.0.1:9200"
ELASTICSEARCH_INDEX = 'samples'
ELASTICSEARCH_TRACE = False

Aleph does not run on port 5000?
He's with URI on port 9200, which is the door?

ELASTICSEARCH_TRACE, because it is False?

Tks ;)

**Comment by jseidl** [comment]
5000 is the Web Interface port. 9200 is elasticsearch which is the
datastore. The aleph parsing daemon doesn't runs in any port

**Comment by danilovazb** [comment]
Hmmm.....okay...

Tks @jseidl 

Someone managed to do some kind of research?
